.github/workflows/run_tests/action.yml

```diff
@@ -19,5 +19,11 @@ runs:
           mpirun -n 3 --oversubscribe tests/test_DMD
           ./tests/test_GreedyCustomSampler
           mpirun -n 3 --oversubscribe tests/test_GreedyCustomSampler
+          cd ${GITHUB_WORKSPACE}/build/examples/prom
+          ./poisson_global_rom -offline -f 1.0 -id 0
+          ./poisson_global_rom -offline -f 1.1 -id 1
+          ./poisson_global_rom -offline -f 1.2 -id 2
+          ./poisson_global_rom -merge -ns 3
```

Comment:
This line is executing one of the examples. It is crashing here on Linux, due to an memory allocation issue. Mac is running fine. This may require updating some settings on Github Actions to increase the memory allocation or changing some build settings in libROM.

Reply:
Hi @kevinhkhuynh @pranav-vempati sorry for being away. Is this working now? The way I solved it on my machine was to increase memory allocation. The settings I have is:

memory=8GB
swap memory=7GB